packages/core/admin/admin/src/content-manager/components/EditViewDataManagerProvider/utils/cleanData.js

```diff
@@ -18,12 +18,7 @@ const cleanData = (retrievedData, currentSchema, componentsSchema) => {
 
       switch (attrType) {
         case 'json':
-          try {
-            cleanedData = JSON.parse(value);
-          } catch (err) {
-            cleanedData = value;
-          }
-
```

Comment:
Could this lead to problems, if I enter invalid JSON? Thinking about something like `{ test }`? This would also cause `JSON.stringify` to throw and might happen, if I copy something from somewhere else, which still contains a variable that needs to be removed before saving it in strapi?

Reply:
You won't be able to save anyway the validation schema will throw an error before we send the data to the API.